bdd/godogs_test.go

```diff
@@ -253,6 +266,30 @@ func (s *store) iRequestEndpoint(path string) error {
 	)
 }
 
+func (s *store) iRequestDeleteDashEndpoint(path string) error {
+	return call(
+		http.MethodDelete,
+		createDashURL(path),
```

Comment:
Can you please use `createURL` function?

Reply:
>if I call DELETE http://localhost:8080/api/apis/httpbin-delete-test will it delete the api-definition?

you can call `DELETE http://localhost:8080/tyk/apis/${API_ID}` - instead of `api/apis`, GW API uses `tyk/apis`. you can find more details [here](https://tyk.io/docs/tyk-gateway-api/) if you wish.

again, thank you for your contribution